### PR TITLE
Add client methods for group role mapping RUD.

### PR DESCRIPTION
**Pull request by galxy25**

### CODE REVIEW

File: group_role_mappings.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	"gopkg.in/h2non/gentleman.v2/plugins/body"
+	"gopkg.in/h2non/gentleman.v2/plugins/url"
+)
+
+const (
+	groupsRealmRoleMappingPath      = "/auth/admin/realms/:realm/groups/:id/role-mappings/realm"
+	groupsRoleMappingByClientIdPath = "/auth/admin/realms/:realm/groups/:group_id/role-mappings/clients/:id"
+)
+
+//GetClientRoleMappings get the realm level roles for the group or error (if any).
```
Comment (willmichael) [inline group_role_mappings.go]: GetGroupRealmRoleMappings

File: group_role_mappings.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+//     "id": "f815fc8a-5eb6-46c1-a454-5fbc8e1c6492",
+//     "name": "offline_access",
+//     "description": "${role_offline-access}",
+//     "composite": false,
+//     "clientRole": false,
+//     "containerId": "4f0f8206-0ec4-4fd6-99eb-4e8c4b986c43"
+//   }
+// ]
+// ```
+func (c *Client) GetGroupRealmRoleMappings(accessToken string, realmName, groupId string) ([]RoleRepresentation, error) {
```
Comment (willmichael) [inline group_role_mappings.go]: any reason to not remove "string" after accessToken?

File: group_role_mappings.go (hunk 1 above)
Comment (galxy25) [reply group_role_mappings.go]: Good catch, updated

**Review by willmichael — APPROVED** [review]

File: group_role_mappings.go (hunk 2 above)
Comment (galxy25) [reply group_role_mappings.go]: nope, updated for both eligible methods

**Review by efabens — APPROVED** [review]

**Pull request merged**